### ISSUE TITLE
Pandora-Iso-App: Update for getting citations from CrossRef

### ISSUE THREAD

**Issue by arunge** [Pandora-IsoMemo/iso-data#3]
Since there was no possibility here to receive a speed for the citation convertion that is applicable in an interactive context, we decided to adjust the code, where the data sources are loaded and written to a database: https://github.com/Pandora-IsoMemo/iso-data
The app only loads pre-formated tables from this database. 

**New goals:** 
- add a "bibtex" citeation style column to the database, such that the app only needs to translate from one style into another style. The translation is only a string operation that should be much faster than loading references from DOI on the fly.
- implement the UI and logic to change the reference style

@jroachell15  I will take over this ticket and develop a solution here.

@isomemo We need to put this as another task into the new task list.

_Originally posted by @arunge in https://github.com/Pandora-IsoMemo/iso-app/issues/14#issuecomment-1156208382_

**Comment by isomemo** [comment]
@arunge @jroachell15 

We will still use Jian's code for the conversion but now instead of directly consulting the (slow) crossref site we will consult our IsoMemo data table where the Bibtex references are going to be stored. These can then be converted into a format selected by the users.

I have created a video that describes my suggested procedure. Please check this carefully: https://youtu.be/49QENRc7zwk

@arunge the bibliographic search is already implemented by Andreas. This returns a DOI from a text reference. What we need is the formatted reference, please see video.

**Comment by arunge** [comment]
> @arunge the bibliographic search is already implemented by Andreas. This returns a DOI from a text reference. 

Yes, this is what I found in the code. I just wanted to take this as an example for possible other parameters and to understand the syntax. Just a reminder for me. :wink:

**Comment by arunge** [comment]
@isomemo With [version 25.10.0](https://github.com/Pandora-IsoMemo/DSSM/pull/288) on our [DSSM beta url](https://isomemoapp.com/app/dssm-beta) I have added new options for exporting citations:

- Select citation columns (1)
- Choose a citation style template (2) or use custom settings (3)
- Export using different printing styles or export types

To support these improvements, I had to update the logic for how citation columns are selected so that both skins — Pandora and Data Networks — now behave consistently. You can freely select and deselect columns in either skin.

If the BibTeX columns input is left empty, citations will not be formatted (for faster export). In this case, only the Reference and DOI columns are included — the output will look as in previous DSSM versions.

Please test this new feature and report any unexpected behavior or errors.
I’m closing this issue here — feel free to open a new one if any problems occur.

<img width="1644" height="1392" alt="Image" src="https://github.com/user-attachments/assets/d238e08d-56e1-4a27-801d-ff746b923134" />

<img width="950" height="578" alt="Image" src="https://github.com/user-attachments/assets/8a8b4c92-96a8-48eb-98ad-3571910316d0" />
